src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+// eslint-disable-next-line
+import CalculatorBtn from './CalculatorBtn';
```

Comment:
- [ ] Please fix the typo error  `'./CalculatorBtn';` should be `'./calculatorBtn';`.

![Capture web_23-11-2021_131837_localhost](https://user-images.githubusercontent.com/61562626/143022926-6889227d-8da9-4e44-9210-eeae5d198e5e.jpeg)

- After update 

![Capture web_23-11-2021_131910_localhost](https://user-images.githubusercontent.com/61562626/143022970-5ff0c6e0-f2f9-4758-893b-fc936b95350b.jpeg)

Reply:
The issue has been resolved